gisad.csv

```diff
@@ -0,0 +1,10 @@
+submitter,fn,covv_virus_name,covv_type,covv_passage,covv_collection_date,covv_location,covv_host,covv_gender,covv_patient_age,covv_patient_status,covv_seq_technology,covv_orig_lab,covv_orig_lab_addr,covv_subm_sample_id,covv_subm_lab,covv_subm_lab_addr,covv_authors,covv_specimen,covv_outbreak,covv_add_host_info,covv_add_location,covv_provider_sample_id,covv_last_vaccinated,covv_treatment,covv_assembly_method,covv_coverage

```

Comment:
dont forget to remove test outputs! And make sure the files are stored someplace in mutant result directory!